SPECS/cassandra-cpp-driver/cassandra-cpp-driver.spec

```diff
@@ -0,0 +1,59 @@
+Summary:        DataStax C/C++ Driver for Apache Cassandra and DataStax Products
+Name:           cassandra-cpp-driver
+Version:        2.16.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://github.com/datastax/cpp-driver
+Source0:        https://github.com/datastax/cpp-driver/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  libuv-devel
+BuildRequires:  openssl-devel
+Requires:       libuv
+
+%description
+A modern, feature-rich, and highly tunable C/C++ client library for Apache
+Cassandra and DataStax Products using Cassandra's native protocol and Cassandra
+Query Language along with extensions for DataStax Products.
+
+%package devel
+Summary:        Development libraries for ${name}
+Group:          Development/Tools
+Requires:       %{name} = 1:%{version}-%{release}
+Requires:       libuv-devel
+
+%description devel
+Development libraries for %{name}
+
+%prep
+%setup -q -n cpp-driver-%{version}
+
+%build
+mkdir -p build
+cd build
+%cmake ..\
+    -DCMAKE_BUILD_TYPE=RELEASE
+%make_build
+
+%install
+cd build
+%make_install
+
+%ldconfig_scriptlets
+
+%files
+%license LICENSE.txt
+%{_libdir}/*.so.2
+%{_libdir}/*.so.2.*
+
+%files devel
+%{_includedir}/*.h
+%{_libdir}/*.so
+%{_libdir}/pkgconfig/*.pc
+
+%changelog
+* Wed Sep 1 2021 Andy Caldwell <andycaldwell@microsoft.com> - 2.16.0-1
+- Original version for CBL-Mariner
```

Comment:
Are we 100% sure it's our original spec and not one taken from a different distribution or from the project owners? The fact we've had the `Epoch:          1` line for a while made me suspicious.

Reply:
That's a question for @bossmc who wrote the spec file for the 1.0 branch.

Reply:
I did write this from scratch, I read the maintainer's version, but that was way over-complicated for Mariner's needs so I just wrote the fairly standard version we're using now.  I couldn't say why I added the `Epoch: 1` maybe I read some best practice somewhere that you should start with one for a first-party package?  If you prefer to start without one and add one as needed I'm happy too.

Reply:
Yes, let's remove `Epoch` from the spec. We actually did a reset of these for 2.0.